dockertest/output/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+API compatibility: on 2016-04-29 output.py got split up into a
+package with four smaller, more maintainable submodules. This init
+allows existing clients to continue importing as a single unit.
+"""
+
+# ARGH. Without this, 'import xceptions' in a submodule fails because
```

Comment:
I would very much like to just kill the xceptions module, there's really no need for it. All of the exceptions are either standard ones or come from autotest.